### DISCUSSION TITLE
Password protect config tab

### DISCUSSION THREAD

**Discussion by wfmcnally86** [blakeblackshear/frigate#6906] [Ask A Question]
I've recently updated to Frigate 12 and love all the new features that have gone into it. I am wondering if there is a way to password protect the config tab. I currently have Authelia restricting web access but I would like to restrict the config tab from other users that have access to the camera feeds. I worry that someone may try and do me favor in the future if something is not working and "help fix it". I have had no success stopping access to individual tabs through Authelia, but I may not be doing it correctly. If this is not possible this may be something to consider for a future release.

**Comment by hagak** [comment]
I am not real familiar with Authelia but it appears they allow you to setup rules such that you can restrict access to certain users based on domain or resources you want to use resources https://www.authelia.com/configuration/security/access-control/#resources and then define what subjects are allowed.

**Comment by didi3r** [comment]
If you are running frigate on docker you can mount the config file with read only access. I mean other users will still be able to see the config but nobody would be able to save the changes made.
